src/Livewire/Settings/LedgerAccounts.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace FluxErp\Livewire\Settings;
+
+use FluxErp\Actions\LedgerAccount\CreateLedgerAccount;
+use FluxErp\Actions\LedgerAccount\DeleteLedgerAccount;
+use FluxErp\Actions\LedgerAccount\UpdateLedgerAccount;
+use FluxErp\Enums\LedgerAccountTypeEnum;
+use FluxErp\Livewire\DataTables\LedgerAccountList;
+use FluxErp\Livewire\Forms\LedgerAccountForm;
+use FluxErp\Models\LedgerAccount;
+use Illuminate\Validation\ValidationException;
+use Spatie\Permission\Exceptions\UnauthorizedException;
+use TeamNiftyGmbH\DataTable\Htmlables\DataTableButton;
+use WireUi\Traits\Actions;
+
+class LedgerAccounts extends LedgerAccountList
+{
+    use Actions;
+
+    protected ?string $includeBefore = 'flux::livewire.settings.ledger-accounts';
+
+    public LedgerAccountForm $ledgerAccount;
+
+    public function getTableActions(): array
+    {
+        return [
+            DataTableButton::make()
+                ->label(__('Create'))
+                ->color('primary')
+                ->icon('plus')
+                ->when(resolve_static(CreateLedgerAccount::class, 'canPerformAction', [false]))
+                ->attributes([
+                    'wire:click' => 'edit()',
+                ]),
+        ];
+    }
+
+    public function getRowActions(): array
+    {
+        return [
+            DataTableButton::make()
+                ->label(__('Edit'))
+                ->icon('pencil')
+                ->color('primary')
+                ->when(resolve_static(UpdateLedgerAccount::class, 'canPerformAction', [false]))
+                ->attributes([
```

Comment:
can just do ->wireClick('edit(record.id)')

Reply:
wie soll ich mit 'wire:confirm.icon.error' => __('wire:confirm.delete', ['model' => __('Ledger Account') - line 55 vorgehen.

Reply:
'wire:click' => 'delete(record.id)' wuerde in wireClick reingetan - leider wurde Modal nicht aufgerufen

Reply:
sorry i mixed delete with edit :) ignore this